bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/metrics/OperationMetrics.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.metrics;
+
+import com.google.api.core.InternalApi;
+import com.google.cloud.bigtable.metrics.BigtableClientMetrics.MetricLevel;
+
+/** Extracted from BigtableAsyncRpc#RpcMetrics, to instrument veneer client. */
+@InternalApi("For internal usage only")
+public class OperationMetrics {
+
+  /** Best effort counter of active RPCs. */
+  public Counter ACTIVE_RPC_COUNTER =
+      BigtableClientMetrics.counter(MetricLevel.Info, "grpc.rpc.active");
+
+  /** Best effort counter of RPCs. */
+  public Meter RPC_METER = BigtableClientMetrics.meter(MetricLevel.Info, "grpc.rpc.performed");
```

Comment:
why are these capitalized? please initialize them in the constructor

Reply:
I tried to follow the existing pattern so I left them capitalized. 

Updated it to be a method in OperationMetrics.